api/envoy/extensions/access_loggers/stats/v3/stats.proto

```diff
@@ -39,13 +41,17 @@ message Config {
     string value_format = 2 [(validate.rules).string = {min_len: 1}];
   }
 
-  // Defines the name and tags of a stat.
-  message Stat {
+  // Defines the common configuration of a stat.
+  message Common {
```

Comment:
Sorry, why change the name?

Reply:
This extension hasn't been released so we still have time ?  If no, I am fine changing back.